src/demos/calcite-block.html

```diff
@@ -141,6 +147,13 @@ <h2>Header + content (collapsible)</h2>
           </calcite-block-section>
         </calcite-block>
 
+        <h2>Header + collapsible + control = no collapsible icon</h2>
+
+        <calcite-block heading="no toggle icon when there is a control" collapsible>
+          <label slot="control">test</label>
```

Comment:
Looks good.
Sand: maybe the demo can use a `calcite-action` with a `...` icon in the control slot? Mostly so as to be instructive.

Here's the SVG if you need it:
```
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><path d="M14 9.05A1.05 1.05 0 1 1 15.05 8 1.05 1.05 0 0 1 14 9.05zm-6 0A1.05 1.05 0 1 1 9.05 8 1.05 1.05 0 0 1 8 9.05zm-6 0A1.05 1.05 0 1 1 3.05 8 1.05 1.05 0 0 1 2 9.05z"/></svg>
```

Reply:
Shouldn't that be a part of the advanced demo?